packages/tools/examples/contourSegmentationRepresentation/index.ts

```diff
@@ -0,0 +1,277 @@
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  setVolumesForViewports,
+  volumeLoader,
+  utilities,
+  geometryLoader,
+  CONSTANTS,
+} from '@cornerstonejs/core';
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+} from '../../../../utils/demo/helpers';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+import contour from './Contour.json';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  SegmentationDisplayTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+  segmentation,
+  ZoomTool,
+  PanTool,
+  StackScrollMouseWheelTool,
+  TrackballRotateTool,
+} = cornerstoneTools;
+const { MouseBindings } = csToolsEnums;
+const { ViewportType, GeometryType } = Enums;
+
+// Define a unique id for the volume
+const volumeName = 'CT_VOLUME_ID'; // Id of the volume less loader prefix
+const volumeLoaderScheme = 'cornerstoneStreamingImageVolume'; // Loader id which defines which volume loader to use
+const volumeId = `${volumeLoaderScheme}:${volumeName}`; // VolumeId with loader id + volume id
+const segmentationId = 'MY_SEGMENTATION_ID';
+const toolGroupId = 'MY_TOOLGROUP_ID';
+const toolGroupId3d = 'MY_TOOLGROUP_ID_3d';
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Contour Segmentation Representation for Volume Viewports',
+  'Here we demonstrate how you can add a contour as a segmentation to a volume viewport. Please note that the contour in this example (for segmentation), is different than the freehandROI tool contour which is an annotation tool. Here, the contour is represented in the 3d space, while the freehandROI tool contour is represented via SVG in the 2d space. So you can render the contour segmentation in the 3d viewport (as seen below) but you cannot render the freehandROI tool contour in the 3d viewport.'
```

Comment:
Consider rewording this to be less freehand specific and more general to 2D tools. I was (at first) left with the impression that the freehand ROI tool was special, but it is not.